Makefile

```diff
@@ -0,0 +1,26 @@
+.DEFAULT_GOAL := help
+
+# determines what "make help" will show
+define PRINT_HELP_PYSCRIPT
+import re, sys
+
+for line in sys.stdin:
+	match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)
+	if match:
+		target, help = match.groups()
+		print("%-20s %s" % (target, help))
+endef
+export PRINT_HELP_PYSCRIPT
+
+################################################################################
+#   General
+
+.PHONY: help 
+
+help: ## Show what this Makefile can do
+	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+
+# may require a "pip install cffconvert" to run
+validate_cff: CITATION.cff
+	cffconvert --validate
+
```

Comment:
more as doc for developers in case they don't remember how to validate those files